projects/netgrif-components-core/src/lib/actions/services/front-action.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable, Injector} from '@angular/core';
+import {FrontActionRegistryService} from "../../registry/front-action-registry.service";
+import {FrontAction} from "../../data-fields/models/changed-fields";
+
+@Injectable()
+export class FrontActionService {
+
+    constructor(protected _injector: Injector,
+                protected _frontActionRegistry: FrontActionRegistryService) {
+
+    }
+
+    public run(frontAction: FrontAction): void {
+        const fn = this._frontActionRegistry.get(frontAction.id)
+        fn.fn(this._injector, frontAction)
```

Comment:
What if `fn` is `undefined` or `null`? There should be check and log that given action is not registered.

Reply:
Done